README.md

```diff
@@ -104,4 +120,11 @@ python3 sparql.py localhost:9090 "SELECT ?subject ?predicate ?object WHERE {?sub
     Now Spark UI should be accessible via `localhost:8080`
 - Query Trident on local:
     `python3 sparql.py 'localhost:9090' "select * where {<http://rdf.freebase.com/ns/m.0d0xs> ?p ?o} limit 1000"`
+    
+- Connect to DAS4 from Uni and home:
+    From home:
+    `ssh -o preferredauthentications=password <VUNET-ID>@ssh.data.vu.nl`
+    `ssh <GROUP-NAME>@fs0.das4.cs.vu.nl`
```

Comment:
You can actually combine the 2 ssh calls into one ssh using this: `ssh -oProxyJump=<VUNET_ID>@ssh.data.vu.nl <GROUP_ID>@fs0.das4.cs.vu.nl` :)